core/dvector.h

```diff
@@ -424,6 +425,17 @@ Error DVector<T>::resize(int p_size) {
 	return OK;
 }
 
+template<class T>
+void DVector<T>::invert() {
+	T temp;
+
+	for(int i=0;i<size()/2;i++) {
+		temp = get(i);
+		set(i, get(size()-i-1));
+		set(size()-i-1, temp);
+	}
```

Comment:
@kjohnson0451 If it is before the loop, you will make `size()/2` less calls to `size()`, since there are calls to it inside the loop as well.